### ISSUE TITLE
findFromRawSql() doesn't accept DISTINCT

### ISSUE THREAD

**Issue by Zheness** [thecodingmachine/tdbm#56]
Hello,

I'm trying to use findFromRawSql with a custom query and instead of having `SELECT table.* FROM` as the doc:

```
You should not put an alias on the main table name, and select its columns using `*`. So the SELECT part of you $sql should look like:

    "SELECT table.* FROM ..."
```

I'm using `SELECT DISTINCT table.* FROM` but I got an error:

```
Exception:
Invalid array. Could not find expression type: array (
  'delim' => ',',
)

LINE: 23
FUNCTION: SQLParser\Node\NodeFactory::toObject()
FILE: /var/www/html/vendor/mouf/magic-query/src/SQLParser/Node/NodeFactory.php 
```

Is the usage of DISTINCT not possible ? If so, maybe change the doc from `should look like` to `must look like` ?

Thanks

**Comment by moufmouf** [comment]
This must be an issue with magic-query... I'm looking into it.

**Comment by moufmouf** [comment]
Actually, this seems unrelated to magic-query. See #66

**Comment by moufmouf** [comment]
Fixed by #66
